examples/website/i3s/helpers/sublayers.js

```diff
@@ -0,0 +1,36 @@
+export function buildSublayersTree(sublayers) {
+  const fullModel = sublayers.find((subtree) => subtree.modelName === 'FullModel');
+  const result = postprocessNode(fullModel);
+  return result;
+}
+
+function postprocessNode(sublayer) {
```

Comment:
Don't you feel like this type of functionality should be provided by the `@loaders.gl/i3s` module? A good "litmus test" is to if anyone using the building layers would need to write this code...

Reply:
I mostly created this code to 
1. deep copy the tree for React
2. filter "preview" layer
3. calculate the number of child nodes for each node (just in case)
It is not necessary for general users of I3SBuildingSceneLayerLoader. For somebody who wants to show "preview" layer it even would be breaking thing

Reply:
Understood. Some JSDoc headers with short explanations would probably be helpful. After all this is an example that should be easy for users to understand.

Reply:
Done